### PR TITLE
List of goods

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/react_goods-selector/)

### CODE REVIEW

**Review by JamesSunderlanddd — APPROVED** [review]
I don't see any issues. Nicely Done!

File: src/App.tsx [hunk 1]
```diff
@@ -14,11 +15,104 @@ const goodsFromServer: string[] = [
...
+
+            {selectedGoods.length ? (
+              <button
+                type="button"
+                onClick={this.clearSelection}
+                className="App__clear"
+              >
+                x
+              </button>
+            ) : ''}
```
Comment (ihor-jpeg) [inline src/App.tsx:85]: you can make a variable that will equal false when the selected goods length is 0. and use && to conditionally render the title

File: src/App.tsx [hunk 2]
```diff
@@ -14,11 +15,104 @@ const goodsFromServer: string[] = [
...
+
+  render() {
+    const { goods, selectedGoods } = this.state;
+
+    return (
+      <div className="App">
+        <div className="App__content">
+          <div className="App__titles">
+            <h1 className="App__title">
+              {this.getSelectedItems()}
```
Comment (ihor-jpeg) [inline src/App.tsx:74]: make a variable for this one

**Review by ihor-jpeg — APPROVED** [review]
good work. let's just fix a few things

**Pull request closed without merging**